merkle/rfc6962/rfc6962.go

```diff
@@ -47,10 +69,24 @@ func (t TreeHasher) HashLeaf(leaf []byte) []byte {
 
 // HashChildren returns the inner Merkle tree node hash of the the two child nodes l and r.
 // The hashed structure is NodeHashPrefix||l||r.
-func (t TreeHasher) HashChildren(l, r []byte) []byte {
+func (t *TreeHasher) HashChildren(l, r []byte) []byte {
 	h := t.New()
 	h.Write([]byte{RFC6962NodeHashPrefix})
 	h.Write(l)
 	h.Write(r)
 	return h.Sum(nil)
 }
+
+// createNullHashes returns a list of empty hashes starting with the hash of an empty
+// tree, all the way down to the hash of an empty leaf.
+func (t *TreeHasher) createNullHashes() [][]byte {
```

Comment:
Woops, I thought I removed this.  It might have creeped back in during a rebase.